plugins/console/spec/consoleConfig.js

```diff
@@ -0,0 +1,17 @@
+var env = require('../../../spec/environment.js');
```

Comment:
In order to get this test to run on travis, you'll need to make an addition to [`scripts/test.js`](https://github.com/angular/protractor/blob/master/scripts/test.js#L39).  Basically change

```js
passingTests.push(
    'node lib/cli.js plugins/timeline/spec/conf.js',
    'node lib/cli.js plugins/ngHint/spec/successConfig.js',
    'node lib/cli.js plugins/accessibility/spec/successConfig.js');
```

to

```js
passingTests.push(
    'node lib/cli.js plugins/timeline/spec/conf.js',
    'node lib/cli.js plugins/ngHint/spec/successConfig.js',
    'node lib/cli.js plugins/accessibility/spec/successConfig.js',
    'node lib/cli.js plugins/console/spec/consoleConfig.js');
```